Tests/SwiftBuildTests/BuildOperationTests.swift

```diff
@@ -1137,7 +1137,7 @@ fileprivate struct BuildOperationTests: CoreBasedTests {
     }
 
     /// Check some cancellation related semantics.
-    @Test
+    @Test(.skipHostOS(.windows, "requires /usr/bin/yes"))
```

Comment:
Oddly this fails for me locally, but not on CI (note #199 Windows build is passing). Maybe due to nondeterministic cancellation timing?